samples/starterKit/.gitignore

```diff
@@ -0,0 +1,85 @@
+aws-config.js
+#### OSX
+*.DS_Store
```

Comment:
Could we remove the sample dir from this project?

Reply:
i think an example dir would be great, something that shows how to use the plugin. But it should be a very minimal example. Even smaller than this. Maybe something like the package.json and an index.js file, with variables and views in the same file, comments?

Reply:
Just made it that way, based on the voxa example in the README